CppCoreGuidelines.md

```diff
@@ -15704,6 +15704,42 @@ Alternatively, we will decide that no change is needed and delete the entry.
 * Speaking of lambdas, what would weigh in on the decision between lambdas and (local?) classes in algorithm calls and other callback scenarios?
 * And speaking of `std::bind`, Stephen T. Lavavej criticizes it so much I'm starting to wonder if it is indeed going to fade away in future. Should lambdas be recommended instead?
 * What to do with leaks out of temporaries? : `p = (s1 + s2).c_str();`
+
+  * Can this example not be fixed in the library? E.g. `char const* c_str() const &; char const* c_str() &&=delete;`. gcc 5.3.1 on Fedora produces an error when given the code:
+
+##### sample code example
+
```

Comment:
Okay, seems to be related to the discussion on line 15708.

I think it might look better if you change the nested bullet point to something slightly less structured, like the following:

    * What to do with leaks out of temporaries? : `p = (s1 + s2).c_str();` For example, perhaps this can be
    fixed in the library - e.g. `char const* c_str() const &; char const* c_str() &&=delete;`. gcc 5.3.1
    produces an error when given the following:

And then your nested code won't get formatted like a header (I think).

How do you feel about that?